fdbserver/Knobs.cpp

```diff
@@ -251,6 +251,16 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs, bool isSimula
 	init( SQLITE_CHUNK_SIZE_PAGES,                             25600 );  // 100MB
 	init( SQLITE_CHUNK_SIZE_PAGES_SIM,                          1024 );  // 4MB
 	init( SQLITE_READER_THREADS,                                  64 );  // number of read threads
+	init( SQLITE_WRITE_WINDOW_SECONDS,                 -1 );
+	init( SQLITE_WRITE_WINDOW_LIMIT,                   -1 );
+	if( randomize && BUGGIFY ) {
+		// Choose an window between .01 and 1.01 seconds.
+		SQLITE_WRITE_WINDOW_SECONDS = 0.01 + deterministicRandom()->random01();
+		// Choose 10k to 50k operations per second
```

Comment:
```suggestion
		// Choose random operations per second
```
This comment was wrong from my previous PR, no need to specify the numbers twice so I just made it more general.